src/routes/+layout.svelte

```diff
@@ -56,7 +56,7 @@
 	 */
 	@custom-media --viewport-sm-up (width >= 576px);
 	@custom-media --viewport-md-up (width >= 768px);
-	@custom-media --viewport-lg-up (width >= 1024px);
+	@custom-media --viewport-lg-up (width >= 1025px);
```

Comment:
Hmm… this doesn't seem right to me.

I don't have an opinion or preference about which direction (`up` vs. `down`) to be _inclusive_ vs. _exclusive_, but I think we need be consistent. So if we want to be _inclusive_ on the `up` side of the breakpoints, then I'd recommend we go with:

```css
@custom-media --viewport-sm-up (width > 576px);
@custom-media --viewport-md-up (width > 768px);
@custom-media --viewport-lg-up (width > 1024px);
@custom-media --viewport-md-down (width <= 1024px);
@custom-media --viewport-sm-down (width <= 768px);
@custom-media --viewport-xs (width <= 576px);
```

Reply:
@kenkunz yes, sure, we can do it like this!